### PR TITLE
[hi] Localize /docs/setup/prodution-environment/tools/

### LINKED ISSUE #45213: [hi] Localize /docs/setup/production-environment/tools/
**This is a Feature Request**

**What would you like to be added**
Localize docs/setup/production-environment/tools/ in Hindi

**Page -**
https://kubernetes.io/docs/setup/production-environment/tools/

### PR DESCRIPTION
**Pull request by ArvindParekh**
Fixes #45213 

Localized the content at [/docs/setup/prodution-environment/tools/](https://kubernetes.io/docs/setup/production-environment/tools/) page to Hindi.

### CODE REVIEW

File: content/hi/docs/setup/production-environment/tools/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "परिनियोजन टूल के साथ कुबेरनेट्स स्थापित करे"
+weight: 30
+no_list: true
+---
+
+अपना स्वयं का उत्पादन कुबेरनेट्स क्लस्टर स्थापित करने के लिए कई विधियाँ और उपकरण है। उदाहरण के लिए:
```
Comment (kundan2707) [inline content/hi/docs/setup/production-environment/tools/_index.md:7]:
```suggestion
अपना स्वयं का प्रोडक्शन कुबेरनेट्स क्लस्टर स्थापित करने के लिए कई विधियाँ और उपकरण है। उदाहरण के लिए:
```

**Review by kundan2707 — COMMENTED** [review]
/lgtm

File: content/hi/docs/setup/production-environment/tools/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+no_list: true
+---
+
+अपना स्वयं का प्रोडक्शन कुबेरनेट्स क्लस्टर स्थापित करने के लिए कई विधियाँ और उपकरण है। उदाहरण के लिए:
+
+- [kubeadm](/docs/setup/production-environment/tools/kubeadm/)
+
+- [kops](https://kops.sigs.k8s.io/): एक स्वचालित क्लस्टर प्रावधान उपकरण। ट्यूटोरियल, सर्वोत्तम प्रथाओं, कॉन्फ़िगरेशन विकल्पों और समुदाय तक पहुंचने की जानकारी के लिए, कृपया [`kOps` वेबसाइट](https://kops.sigs.k8s.io/) देखें।
+
+- [kubespray](https://kubespray.io/): जेनेरिक ओएस/कुबेरनेट्स क्लस्टर कॉन्फ़िगरेशन प्रबंधन कार्यों के लिए [अन्सिबल](https://docs.ansible.com/) प्लेबुकस, [इन्वेंट्री](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible.md#inventory), प्रोविजनिंग टूल और डोमेन ज्ञान की एक संरचना। आप स्लैक चैनल [#kubespray](https://kubernetes.slack.com/messages/kubespray/) पर समुदाय को संपर्क कर सकते है।
```
Comment (divya-mohan0209) [inline content/hi/docs/setup/production-environment/tools/_index.md:13]:
```suggestion
- [kubespray](https://kubespray.io/): सामान्य OS /कुबेरनेट्स क्लस्टर कॉन्फ़िगरेशन प्रबंधन कार्यों के लिए [अन्सिबल](https://docs.ansible.com/) प्लेबुकस, [इन्वेंट्री](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible.md#inventory), प्रोविजनिंग टूल और डोमेन ज्ञान की एक संरचना। आप स्लैक चैनल [#kubespray](https://kubernetes.slack.com/messages/kubespray/) पर समुदाय को संपर्क कर सकते है।
```

**Review by divya-mohan0209 — DISMISSED** [review]
Since OS is the acronym, I would recommend not transliterating it.

Additionally, generic = सामान्य

**Comment by ArvindParekh** [comment]
Got it, thanks for the suggestions, Divya! I've made the changes 👍🏻

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/lgtm

**Pull request merged**